salt/grains/core.py

```diff
@@ -1769,8 +1769,16 @@ def os_data():
                                     comps[3].replace('(', '').replace(')', '')
                 elif os.path.isfile('/etc/centos-release'):
                     log.trace('Parsing distrib info from /etc/centos-release')
-                    # CentOS Linux
-                    grains['lsb_distrib_id'] = 'CentOS'
+                    # Maybe CentOS Linux; could also be SUSE Expanded Support.
+                    # SUSE ES has both, centos-release and redhat-release.
+                    if os.path.isfile('/etc/redhat-release'):
+                        with salt.utils.fopen('/etc/redhat-release') as ifile:
```

Comment:
This should be `salt.utils.files.fopen()` in develop now.

Reply:
Thanks for catching this. Fixed it and squashed the commits.